randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -426,6 +453,10 @@ Extra - room_id: [14]
   > Door to Central Shaft
       Trivial
 
+> Other to Conceiled Ceiling; Heals? False
+  * Layers: default
+  * Open Passage to Conceiled Ceiling/Lock - Other to Dessgeega Dormitory
+
```

Comment:
This connection does not exist in the current patcher and it is currently not planned. Please remove this dock and connection between `Dessgeega Dormitory -> Conceiled Ceiling`.

https://discord.com/channels/119665192110915584/1199873303892992020/1207705329035649034